### PR TITLE
New background

### PR DESCRIPTION
**Pull request by ghost**
## Describe the change

I have extended the existing background with more information and included diagrams and video links for various sections. Both Eugen and Caroline have read this page and their feedback has been implemented.

You can view the background [here](https://greengypsy-staging-openms.readthedocs.io/en/new_background/introduction/extended-background.html).

## PR checklist

- [x] I have added description of the change I'm proposing in the OpenMS Documentation.
- [x] I have read and followed [OpenMS Documentation Contributing guidelines](CONTRIBUTING.md).
- [ ] I have attached a screenshot of the relevant area after this change.
- [ ] `CHANGELOG.md` is updated.
- [ ] I have added my name in [CONTRIBUTING.md](CONTRIBUTING.md#openms-documentation-contributors).

### CODE REVIEW

File: .vscode/targets.log [hunk 1]
```diff
@@ -1,258 +1,4 @@
 make all --print-data-base --no-builtin-variables --no-builtin-rules --question
-# GNU Make 3.81
-# Copyright (C) 2006  Free Software Foundation, Inc.
-# This is free software; see the source for copying conditions.
-# There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
-# PARTICULAR PURPOSE.
 
-# This program built for i386-apple-darwin11.3.0
- 
+You have not agreed to the Xcode license agreements, please run 'sudo xcodebuild -license' from within a Terminal window to review and agree to the Xcode license agreements.
```
Comment (timosachsenberg) [inline .vscode/targets.log]: please don't check in the .vscode log files. also above

Comment (ghost) [reply .vscode/targets.log]: I don't know how to fix this...

Comment (timosachsenberg) [reply .vscode/targets.log]: https://stackoverflow.com/questions/39361115/git-ignore-accidentally-commited-file

Comment (ghost) [reply .vscode/targets.log]: done I think

**Review by timosachsenberg — APPROVED** [review]

**Pull request merged**